uqcsbot/api.py

```diff
@@ -305,5 +326,130 @@ def _on_group_joined(self, evt):
     def _on_group_left(self, evt):
         self._on_channel_deleted(evt)
 
-class MembersWrapper(object):
-    pass
+
+class UsersWrapper(object):
```

Comment:
if changing to `users` from `members` (which I agree with) can you change it everywhere its mentioned in the repo pls

Reply:
i.e. `member_ids`

Reply:
I kind of feel like the channel one should stay as is tbh - it's talking about the members of a generic "conversation", whereas the user data store stores info about all users

Reply:
sgtm